tests/test_pipeline.py

```diff
@@ -174,4 +174,23 @@ def test_pipeline_logging(tmp_path):
     with open(logfile, "r", encoding="utf-8") as f:
         firstline = f.readline().rstrip()
     assert "Start running matchms pipeline" in firstline
-    
\ No newline at end of file
+    
+
+def test_FingerprintSimilarity_pipeline():
+    pipeline = Pipeline()
+    pipeline.query_files = spectrums_file_msp
```

Comment:
```suggestion
    pytest.importorskip("rdkit")
    pipeline = Pipeline()
    pipeline.query_files = spectrums_file_msp
```

Reply:
I think something like this should do the trick to skip this test if rdkit is not installed (which is also what we did for other rdkit-reliant tests).